conf.json

```diff
@@ -1,9 +1,9 @@
-{ settings: 
+{ "settings": 
 	[
 		{
-		  "src_table": "info",
-		  "dest_table": "info",
-		  "column_mapping": {"ID_": "id", "DURATION_": "start_time"},
+		  "src_table": "Camunda_dump_example_csv",
```

Comment:
Just a question from an outsider: why aren't you using camelCase for JSON properties?

See also [Google JSON style guide](https://google.github.io/styleguide/jsoncstyleguide.xml).

Of course if there is a good reason, ignore this comment.